csunplugged/topics/content/en/image-representation/unit-plan/unit-plan.md

```diff
@@ -0,0 +1,94 @@
+A black and white image, where each pixel can be only either black or white, is much simpler to represent using digits than a full length, colour, and high-definition film is!
+In this unit students are introduced to simple examples of how they can use digits to create images, how the number of colours an image can include is based on the number of bits used to store it, and how images can be compressed so they take up less memory space and computers can process them faster.
+
+## Digital Technologies | Data Representation
+
+All data on computers is represented with digits, and using these digits to represent other types of data and information is a core concept of Computer Science.
+At this point you have probably noticed the words **represent** and **representation** are being used a lot, so let's look at exactly what we mean by this.
+
+{image file-path="img/topics/binary-picture-showing-bits.png" alt="If we look closely at an image on a computer screen we can see it is made up of a grid of tiny dots called pixels. The colour of each of these is stored inside a computer using binary."}
+
+The type of information a set of digits **represents** changes the way we interpret it.
+For example, if a set of digits represents an image then we interpret those digits as the colours of different pixels.
+If it is meant to be text on the other hand, we interpret it as letters.
+The ability for computers to represent multiple types of information using just two states (represented with binary digits) is one of the things that makes them so powerful.
+
+Just like learning about the binary number system, exploring image representation exposes students to the Computational Thinking concepts of Abstraction and Decomposition.
+Students learn to break down images into pixels and then to digits, and how to move back up from digits, to pixels, to images.
+
+## Digital Technologies | Algorithms
+
+When pixels are used to represent images, the process we go through to convert digits into these pixels is a type of Algorithm.
+There are many different algorithms computers use to do this, and the one they use depends on things like the file type of the images being displayed, for example if it is a picture like a .png or .jpeg, or a video file like an .mp4 or .avi; or what resolution the image is.
+We also use Algorithms to compress images to smaller file sizes, which means they use less memory, can be processed more quickly, and are faster to download.
+Through lerning about image representation students practice algorithmic thinking, pattern matching, and abstract thinking.
+
+## Vocabulary Explained
+
+### Pixel
+
+Computer screens are divided into a grid of tiny coloured squares, which are called pixels.
+The colour of each pixel can be set by a computer and they are used to display images on a computer screen (and printed images).
+The word pixel is an abbreviation of *picture element*.
```

Comment:
I don't really get having a glossary definition as well as this section (and I have the same issue regarding printers)

Reply:
I've removed printers/printed images in each place, I think you're right that it wasn't correct to have that in there.

I think having the definition written here as well as having a glossary definition is useful because it highlights it as one of the important words/concepts in this topic, and it's consistent with how other unit plans include the definitions.